### PR TITLE
feat: add mekong network option

### PR DESCRIPTION
**Pull request by nflaig**
Add network option `--network mekong` to join the new public electra devnet https://mekong.ethpandaops.io/.

### CODE REVIEW

File: packages/config/src/chainConfig/networks/mekong.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // Capella
+  CAPELLA_FORK_VERSION: b("0x40000000"),
+  CAPELLA_FORK_EPOCH: 0,
+  // Deneb
+  DENEB_FORK_VERSION: b("0x50637624"),
+  DENEB_FORK_EPOCH: 0,
+  // Electra
+  ELECTRA_FORK_VERSION: b("0x60637624"),
+  ELECTRA_FORK_EPOCH: 256,
+
```
Comment (ensi321) [inline packages/config/src/chainConfig/networks/mekong.ts:38]: It looks like `SECONDS_PER_ETH1_BLOCK` is 12 in mekong. Maybe it's a typo in ethpandaops's config?

**Review by ensi321 — APPROVED** [review]
lgtm

**Comment by philknows** [comment]
We should probably include this with the RC for v1.23 @wemeetagain ? We likely won't be pushing a v1.24 for at least a couple weeks.

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.23.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.23.0) :tada:

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.24.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.24.0) :tada:

**Pull request merged**